resources/js/store/modules/teams/actions.js

```diff
@@ -352,15 +349,6 @@ export const actions = {
                     position: 'bottom-right'
                 });
             }
-
-            else
-            {
-                Vue.$vToastify.error({
-                    title: failTitle,
-                    body: failBody,
-                    position: 'bottom-right'
-                });
-            }
```

Comment:
Why are we removing this team not found by identifier notification? The user will need a response that a team does not exist

Reply:
I actually changed the existing behavior for the 'team not found' validation. Instead of having it as a separate validation within method code, it's now handled directly in `JoinTeamRequest`. The error message will appear as a validation error, similar to other validation errors.
We use this request both in API and web TeamsController.